java/test/jmri/jmrit/display/layoutEditor/load/LayoutEditorTest.xml

```diff
@@ -1,298 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please move this file to the test/JMRI/jmrix/display/layoutEditor/valid directory.  We still need to make sure this file loads.